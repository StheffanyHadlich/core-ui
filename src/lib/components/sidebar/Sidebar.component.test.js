import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/sidebar/__snapshots__"
});
