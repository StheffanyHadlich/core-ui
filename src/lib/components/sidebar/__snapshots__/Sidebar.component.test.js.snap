// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Sidebar Default 1`] = `
<div>
  <h3>
    Sidebar docked
  </h3>
  <div
    style={
      Object {
        "width": "55px",
      }
    }
  >
    <div
      className="sc-sidebar sc-bdVaJa gguJuH"
    >
      <div
        className="sc-sidebar-item sc-bwzfXH jBzhYZ"
        onClick={[Function]}
        title="Dashboard"
      >
        <div
          className="sc-ifAKCX irnTxV"
        >
          <i
            className="fas fa-tachometer-alt"
          />
        </div>
        <div
          className="sc-bxivhb dwOGvF"
        />
      </div>
      <div
        className="sc-sidebar-item sc-bwzfXH gcwdVF"
        onClick={[Function]}
        title="Servers"
      >
        <div
          className="sc-ifAKCX irnTxV"
        >
          <i
            className="fas fa-server"
          />
        </div>
      </div>
      <div
        className="sc-sidebar-item sc-bwzfXH gcwdVF"
        onClick={[Function]}
        title="Disks"
      >
        <div
          className="sc-ifAKCX irnTxV"
        >
          <i
            className="fas fa-hdd"
          />
        </div>
      </div>
    </div>
  </div>
  <h3>
    Sidebar expanded
  </h3>
  <div
    style={
      Object {
        "width": "150px",
      }
    }
  >
    <div
      className="sc-sidebar sc-bdVaJa kxFxIY"
    >
      <div
        className="sc-sidebar-item sc-bwzfXH jBzhYZ"
        onClick={[Function]}
        title="Dashboard"
      >
        <div
          className="sc-ifAKCX irnTxV"
        >
          <i
            className="fas fa-tachometer-alt"
          />
        </div>
        <div
          className="sc-htpNat jyYvCB"
        >
          Dashboard
        </div>
        <div
          className="sc-bxivhb dwOGvF"
        />
      </div>
      <div
        className="sc-sidebar-item sc-bwzfXH gcwdVF"
        onClick={[Function]}
        title="Servers"
      >
        <div
          className="sc-ifAKCX irnTxV"
        >
          <i
            className="fas fa-server"
          />
        </div>
        <div
          className="sc-htpNat jyYvCB"
        >
          Servers
        </div>
      </div>
      <div
        className="sc-sidebar-item sc-bwzfXH gcwdVF"
        onClick={[Function]}
        title="Disks"
      >
        <div
          className="sc-ifAKCX irnTxV"
        >
          <i
            className="fas fa-hdd"
          />
        </div>
        <div
          className="sc-htpNat jyYvCB"
        >
          Disks
        </div>
      </div>
    </div>
  </div>
</div>
`;
